HaikuPorter/ShellScriptlets.py

```diff
@@ -676,8 +676,18 @@ def getShellVariableSetters(shellVariables):
 
 	local objcopy=$(getTargetArchitectureCommand objcopy)
 	local strip=$(getTargetArchitectureCommand strip)
+	local xres=$(getTargetArchitectureCommand xres)
```

Comment:
You don't need this because there isn't an xres-x86 (the same binary works for all architectures directly).